pir/cpp/server_test.cpp

```diff
@@ -106,20 +109,51 @@ TEST_F(PIRServerTest, TestProcessRequest) {
   auto payload = PIRPayload::Load(query, gal_keys);
 
   auto result_or = server_->ProcessRequest(payload);
-  ASSERT_THAT(result_or.ok(), IsTrue());
+  ASSERT_THAT(result_or.ok(), IsTrue())
+      << "Error: " << result_or.status().ToString();
   auto result = result_or.ValueOrDie();
   ASSERT_THAT(result.Get(), SizeIs(1));
 
   Plaintext result_pt;
   decryptor_->decrypt(result.Get()[0], result_pt);
   auto encoder = server_->Context()->Encoder();
   ASSERT_THAT(encoder->decode_int64(result_pt),
-              Eq(db_[desired_index] * next_power_two(DB_SIZE)));
+              Eq(db_[desired_index] * next_power_two(db_size_)));
+}
+
+TEST_F(PIRServerTest, TestProcessRequest_MultiCT) {
+  SetUpDB(5000);
```

Comment:
Suggestion: `2 * DEFAULT_POLY_MODULUS_DEGREE` or something similar.

Reply:
I'm actually thinking that perhaps the opposite in this case would be best: define the poly mod degree for this test. Makes things more static and obvious for testing.

Reply:
Sounds good, but two values should not be hardcoded without being next to each other; otherwise someone could modify one and the test could become a "SingleCT" instead of "MultiCT" quite easily.

Reply:
Then it would fail pretty obviously :)
In any case, this is now set explicitly in this file. I'd love to optimize the settings so that it's faster for the oblivious expansion tests, but I haven't found a combination that works yet for anything smaller than 4096.